### PR TITLE
Add some random chance commands.

### PR DESCRIPTION
**Pull request by hkwu**
:game_die: :game_die: :game_die: :game_die: :game_die:

### CODE REVIEW

File: commands/Fun/choice.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-exports.run = (client, msg, choices) => {
+exports.run = (client, msg, ... [choices]) => {
```
Comment (hkwu) [inline commands/Fun/choice.js]: Ack, choices should actually be just one array.

`!choice Red|Black|Orange red` => `['Red', 'Black', 'Orange red']`.

File: commands/Fun/card.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const suits = ['♠️', '♦', '♥️', '♠️'];
+const ranks = ['A', '2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K'];
+
+exports.run = (client, msg, [num]) => {
+  const numCards = num === undefined ? 1 : num;
```
Comment (CyberiumShadow) [inline commands/Fun/card.js:5]: you could do a !(num) I think

File: commands/Fun/card.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  if (numCards < 1 || numCards > 10) {
+    msg.channel.sendMessage('Number of cards should be between 1 and 10.');
+
+    return;
+  }
+
+  const lines = [];
+
+  for (let i = 0; i < numCards; ++i) {
```
Comment (CyberiumShadow) [inline commands/Fun/card.js:15]: Should be i++ but following ESlint rules, Possibly changed to i =+ 1
Run it through the updated linter rules.

**Review by CyberiumShadow — CHANGES_REQUESTED** [review]
Possible Changes to consider

File: commands/Fun/choice.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+exports.run = (client, msg, choices) => {
```
Comment (CyberiumShadow) [inline commands/Fun/choice.js:1]: `exports.run = (client, msg, ... [choices]) => {`

Array of choices allowing for multiple args? maybe

**Review by CyberiumShadow — CHANGES_REQUESTED** [review]

**Review by CyberiumShadow — APPROVED** [review]

**Review by CyberiumShadow — CHANGES_REQUESTED** [review]
Ignore that approval. Just see if you can make those changes I outlined.

File: commands/Fun/card.js (hunk 2 above)
Comment (hkwu) [reply commands/Fun/card.js:5]: Had to distinguish between `0` and `undefined` cause I wanted it to trigger

``` js
msg.channel.sendMessage('Number of cards should be between 1 and 10.');
```

I would've used the built in usage min and max but it doesn't seem to work for optional args.

File: commands/Fun/card.js (hunk 3 above)
Comment (hkwu) [reply commands/Fun/card.js:15]: Hmm, thought I changed it so plusplus is ok.

File: commands/Fun/choice.js (hunk 4 above)
Comment (hkwu) [reply commands/Fun/choice.js:1]: How I understood it is that the user is going to type in

```
!choice Option 1|Option 2|Option3
```

and the framework passes in

``` js
['Option 1', 'Option 2', 'Option 3']
```

as the `choices` arg. Wouldn't that make it just take in the first option only?

Comment (CyberiumShadow) [reply commands/Fun/choice.js:1]: I'll test it tonight

**Review by CyberiumShadow — APPROVED** [review]

**Comment by CyberiumShadow** [comment]
Do you still want this to be merged @hkwu

**Comment by hkwu** [comment]
@CyberiumShadow sure.

**Pull request merged**